test/services/cert/certs_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'katello_test_helper'
+require 'support/candlepin/owner_support'
+
+module Katello
+  class CertsTest < ActiveSupport::TestCase
+    def setup
+      VCR.insert_cassette('lib/tasks/verify_ueber_cert')
+      @org = get_organization
+      Resources::Candlepin::Owner.create(@org.label, @org.name)
+    end
+
+    def teardown
+      Resources::Candlepin::Owner.destroy(@org.label)
+      VCR.eject_cassette
+    end
+
+    def test_verify_ueber_cert
+      Setting.stubs(:[]).with(:ssl_ca_file).returns(File.join("#{Rails.root}", "/test/services/cert/helpers/ca.crt"))
```

Comment:
@jlsherrill @stbenjam I keep getting this error on a test run:
```  1) Error:
Katello::CertsTest#test_verify_ueber_cert:
OpenSSL::X509::StoreError: system lib
    /home/vagrant/katello/app/services/cert/certs.rb:22:in `add_file'
    /home/vagrant/katello/app/services/cert/certs.rb:22:in `verify_ueber_cert'
    /home/vagrant/katello/test/services/cert/certs_test.rb:20:in `test_verify_ueber_cert'

```

any ideas why that is? I've even tried the hardcoded full path